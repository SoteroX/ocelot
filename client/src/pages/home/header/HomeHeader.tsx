import * as React from "react";
import { Button } from "semantic-ui-react";
import { NavLink } from "react-router-dom";

import Navbar from "../../../components/navbar/Navbar";

// Images
// import productPic from "../../../images/landing/product2.jpg";
import headerBg from "../../../images/landing/header-bg2.jpg";

const HomeHeader: React.FC = () => {
  return (
    <div
      className="flex flex-justify-center flex-align-items-center home-header-section"
      style={{
        height: "100%",
        // backgroundImage: `url(${headerBg})`
        backgroundImage: `url(${headerBg})`
      }}
    >
      <div style={{ height: "100%", width: "100%" }}>
        <Navbar />
        <div className="flex home-header-content-container">
          <div className="flex  flex-column home-header-content">
            <h1 className="home-header-title">Ocelot</h1>
            <h1 className="home-header-subtitle">
              Build products without the need for multiple apps
            </h1>
            <p className="home-header-text">
              Ocelot lets you to keep track of projects issues and tasks while
              also allowing you to chat with project members.
            </p>
            <div>
              <NavLink to="/signin">
                <Button>Sign in</Button>
              </NavLink>
              <NavLink to="/signup">
                <Button>Sign up</Button>
              </NavLink>
            </div>
          </div>
          {/* <div className="home-header-img-container">
            <img className="home-header-img" src={productPic} />
          </div> */}
        </div>
      </div>
    </div>
  );
};

export default HomeHeader;
