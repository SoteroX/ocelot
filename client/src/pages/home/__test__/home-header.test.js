import React from "react";
import { shallow } from "enzyme";
import HomeHeader from "../header/HomeHeader";

it("should render home page", () => {
  expect(shallow(<HomeHeader />)).toMatchSnapshot();
});
