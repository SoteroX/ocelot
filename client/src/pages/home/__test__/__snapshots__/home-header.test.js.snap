// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home page 1`] = `
<div
  className="flex flex-justify-center flex-align-items-center home-header-section"
  style={
    Object {
      "backgroundImage": "url([object Object])",
      "height": "100%",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <Navbar />
    <div
      className="flex home-header-content-container"
    >
      <div
        className="flex  flex-column home-header-content"
      >
        <h1
          className="home-header-title"
        >
          Ocelot
        </h1>
        <h1
          className="home-header-subtitle"
        >
          Build products without the need for multiple apps
        </h1>
        <p
          className="home-header-text"
        >
          Ocelot lets you to keep track of projects issues and tasks while also allowing you to chat with project members.
        </p>
        <div>
          <NavLink
            to="/signin"
          >
            <Button
              as="button"
            >
              Sign in
            </Button>
          </NavLink>
          <NavLink
            to="/signup"
          >
            <Button
              as="button"
            >
              Sign up
            </Button>
          </NavLink>
        </div>
      </div>
    </div>
  </div>
</div>
`;
